app/xl_NewFileByTemplate.py

```diff
@@ -73,7 +73,7 @@ def SetUp(xlApp):
 			ws = xlApp.Sheets(tempSheetName)
 			ws.Activate
 	        
-			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName[i]) + ".xlsx")
+			xlApp.ActiveWorkbook.SaveAs(newFilePath + "\\" + str(newFileName) + ".xlsx")
 			xlApp.ActiveWorkbook.Close(False)
 			xlApp.screenUpdating = True
 			Marshal.ReleaseComObject(ws)
```

Comment:
go for it. you submitted this PR. prove to me that what you are offering is a better solution that what is there already. You will have to provide some sort of working proof, since I can't merge it in without testing first.